src/drive/web/modules/drive/AddMenu/AddMenu.jsx

```diff
@@ -0,0 +1,86 @@
+import React, { useMemo, useContext } from 'react'
+import { useSelector } from 'react-redux'
+
+import { SharingContext } from 'cozy-sharing'
+import { isMobileApp } from 'cozy-device-helper'
+import useBreakpoints from 'cozy-ui/transpiled/react/hooks/useBreakpoints'
+import { useI18n } from 'cozy-ui/transpiled/react/I18n'
+import ActionMenu from 'cozy-ui/transpiled/react/ActionMenu'
+import Typography from 'cozy-ui/transpiled/react/Typography'
+
+import { getCurrentFolderId } from 'drive/web/modules/selectors'
+import ScanWrapper from 'drive/web/modules/drive/Toolbar/components/ScanWrapper'
+import AddFolderItem from 'drive/web/modules/drive/Toolbar/components/AddFolderItem'
+import CreateNoteItem from 'drive/web/modules/drive/Toolbar/components/CreateNoteItem'
+import CreateShortcut from 'drive/web/modules/drive/Toolbar/components/CreateShortcut'
+import UploadItem from 'drive/web/modules/drive/Toolbar/components/UploadItem'
+import StartScanner from 'drive/web/modules/drive/Toolbar/components/StartScanner'
+
+export const ActionMenuContent = ({
+  isDisabled,
+  canCreateFolder,
+  canUpload,
+  hasWriteAccess
+}) => {
+  const { t } = useI18n()
+  const { isMobile } = useBreakpoints()
+
+  return (
+    <>
+      {isMobile && (
+        <>
+          <Typography variant="h6" className="u-p-1">
+            {t('toolbar.menu_add')}
+          </Typography>
+          <hr />
+        </>
+      )}
+      {canCreateFolder && hasWriteAccess && <AddFolderItem />}
+      {hasWriteAccess && <CreateNoteItem />}
+      {hasWriteAccess && <CreateShortcut />}
+      {canUpload && hasWriteAccess && <hr />}
+      {canUpload && hasWriteAccess && <UploadItem disabled={isDisabled} />}
```

Comment:
Before we had:  
```
{!isShared &&
          canUpload &&
          hasWriteAccess && <UploadButtonItem disabled={isDisabled} />}
```

Now we don't test anymore the isShared. Is is wanted?

Reply:
Oui, mais il y a une question en suspens sur le sujet...

Reply:
Verdict : on le déplace à côté du MoreMenu et la notion `isShared` disparaît (on affiche toujours la même chose que ce soit partagé ou non)